### PR TITLE
Feature: Add GlobalHeader

### PR DESCRIPTION
**Pull request by andrewchough**
# Describe your PR

Related to https://github.com/Rebuild-Black-Business/RBB-Website/issues/32

## Pages/Interfaces that will change
- Every page will now have the GlobalHeader component on the top of the page.

### Screenshots / video of changes

**Web**
Screen Shot 2020-06-06 at 9 13 36 PM

**Mobile**
Screen Shot 2020-06-06 at 9 18 38 PM

### Additional notes
- Added a comment regarding setting up breakpoints. Happy to discuss

### CODE REVIEW

File: src/gatsby-plugin-chakra-ui/theme.js [hunk 1]
```diff
@@ -20,6 +20,9 @@ const customTheme = {
     '2xl': '2rem',
     '3xl': '2.5rem',
   },
+  // Mobile views were not covered under default 'sm' breakpoint
+  // Going to just write in mock sizes, but open to changing them
+  breakpoints: ['16em', '48em', '64em', '80em'],
```
Comment (mbifulco) [inline src/gatsby-plugin-chakra-ui/theme.js:25]: This might be a good one to ask about in design channels if you haven't already

**Comment by magnificode** [comment]
@andrewchough this looks _excellent_ thank you so much for getting this done! In the interest of time, I have a couple of a11y issues that I've expressed in this issue: #57 - If you're able to tackle them that's awesome, otherwise we'll let another contributor grab that issue. But overall this is solid. Thank you!

File: src/gatsby-plugin-chakra-ui/theme.js (hunk 1 above)
Comment (magnificode) [reply src/gatsby-plugin-chakra-ui/theme.js:25]: @mbifulco I think we're cool to just use the ol' create a breakpoint once things start looking broken, methodology for now! Just to keep things rolling. We can refactor to standardize breakpoints if we need.

**Comment by andrewchough** [comment]
> @andrewchough this looks _excellent_ thank you so much for getting this done! In the interest of time, I have a couple of a11y issues that I've expressed in this issue: #57 - If you're able to tackle them that's awesome, otherwise we'll let another contributor grab that issue. But overall this is solid. Thank you!

I can take a crack at this! Thanks for setting this ticket up for me 😃

**Pull request merged**